datascience/tables.py

```diff
@@ -1795,6 +1795,16 @@ def bin(self, *columns, **vargs):
             binned[label + ' ' + tag] = np.append(counts, 0)
         return binned
 
+    def move_column(self, label, index):
+        """Returns a new table with specified column moved to the specified column index."""
+        table = type(self)()
+        col_order = list(self._columns)
+        label = col_order.index(self._as_label(label))
+        col_order.insert(index, col_order.pop(label))
```

Comment:
This has two side-effecting operations on `col_order` in the same statement, which seems unnecessarily tricky.  I suggest rewriting as `col = col_order.pop(label)` then `col_order.insert(index, col)`.  Also I'd find this clearer as `label_idx = col_order.index(self._as_label(label))` and using `label_idx` later, rather than overwriting `label` with a value of a different type.